add login paragraph (#1)
ShopToken/frontend-sdk-documentation: README.md

By default the launcher button is hidden and retailer must either implement `shoptoken.showWidget()`
or set `shoptoken.setConfig({showLauncherButton: false})` to show the launcher button

### Log in

> You can login the user with `shoptoken.login()`. It will first attempt to log in with stored shoptoken jwt tokens. In case this is not successful it will try SSO login using the user account from the retailer platform.

Reviewer: ```suggestion
You can login the user with `shoptoken.autoLogin()`. It will first attempt to log in with stored shoptoken jwt tokens. In case this is not successful it will try SSO login using the user account from the retailer platform.
```

> This method is called automatically when web-js is loaded. Explicit call to this method is needed when a user login to the retailer platform. If this do not cause web-js to be reloaded we need to call `shoptoken.login()` to force shoptoken login.

Reviewer: ```suggestion
This method is called automatically when web-js is loaded. Explicit call to this method may be needed when a user login to the retailer platform and web-js is already loaded.
```
